django/webapp/urls.py

```diff
@@ -15,8 +15,28 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="LMVP (Lightweight Model Versioning Platform) API",
+        default_version='v1',
+        description="Open source neural network versioning system that separates model management and training operations",
+        terms_of_service="(To Be Added)",
+        contact=openapi.Contact(email="hello@umassdsc.com"),
+        license=openapi.License(name="BSD-3-Clause License"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', include('lmvpinterface.urls')),
+    path('lmvp/', include('lmvpinterface.urls')),
```

Comment:
I would prefer that we didn't add this redirection if it's possible. Unless there's a specific conflict you're trying to avoid here, the URLs from the `lmvp` app should be at the root. I agree that auth urls should be separate though.
```suggestion
    path('', include('lmvpinterface.urls')),
```

Reply:
Noted. I'll just adjust the order of the URLs, to prevent the conflict.